_layouts/default.html

```diff
@@ -18,6 +18,12 @@
     <li class="nav-item"><a class="nav-link" href="software.html">Get it</a></li>
     <li class="nav-item"><a class="nav-link" href="https://proxy.jmap.io">Try It</a></li>
 </ul>
+<div id="humberger-menu">
+    <input type="checkbox" />
+    <span></span>
+    <span></span>
+    <span></span>
+</div>
```

Comment:
Since the elements are not used without JavaScript, they should be added by the JavaScript.

Reply:
Done